baggers.js

```diff
@@ -16617,6 +16617,63 @@ var data = {
         mail: 'llys@palo-it.com',
       },
     },
+    {
+      since: '2018-01-01',
+      name: 'Francois-Guillaume Ribreau',
+      bio: 'Full Stack CTO, Startup Advisor, Consultant, Hacker, Maker',
+      picture:
+        'https://media.licdn.com/dms/image/C4D03AQFYw_-F2gSwlg/profile-displayphoto-shrink_200_200/0?e=1560988800&v=beta&t=Doqc2flLGqxUlCWNzWEGpALQ7HGa2kBqTbiVu1Cqn2Q',
+      websites: [
+        {
+          name: 'Site',
```

Comment:
s/Site/Web according to https://github.com/brownbaglunch/bblfr_data#websites

Reply:
fixed :+1: